CHANGELOG.md

```diff
@@ -0,0 +1,98 @@
+
+### Removed
+
+- `solver_parameter` from the model, it now fulfills `solver` and `preprocessing_first_solution_strategy` parameters
+- OR-Tools logs within sheduling heuristic [#28](https://github.com/Mapotempo/optimizer-api/pull/28)
+- Relation id [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Vehicle sort by day index [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- `overall_duration` parameter from API as it is not functional [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+
+### Fixed
+
+- Resolution time when using split_independent or max_split procedures
+- Alternative activities with empties and fills
+- Unreachable points detection
+- Shift computation for sheduling problems
+- Sheduling heuristic now rejects shipments and rests [#28](https://github.com/Mapotempo/optimizer-api/pull/28)
+- Reject empty problems [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Negative time limits [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Expand multi_trips only once [#36](https://github.com/Mapotempo/optimizer-api/pull/36)
+- Remove data from already started jobs [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Clean interrupted working jobs at startup [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Error when delete operation is invoked immedialty after get [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Coerce `first_solution_strategy` into array [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Avoid unecessary route generation [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Correclty handle consecutive delete operations [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
+- Solution returned from cache [#53](https://github.com/Mapotempo/optimizer-api/pull/53)
+- Returned http error codes [#62](https://github.com/Mapotempo/optimizer-api/pull/62)
+- Check distance method to separate time and distance logics [#65](https://github.com/Mapotempo/optimizer-api/pull/65)
+- Split by skills was generating unexpected sub-problems [#47](https://github.com/Mapotempo/optimizer-api/pull/47)
+- Remove duplicated empty routes for the same vehicle [#50](https://github.com/Mapotempo/optimizer-api/pull/50)
+- Infinite loop due to impossible split within `dichotomous` [#67](https://github.com/Mapotempo/optimizer-api/pull/67)
+- Avoid depot duplication [#72](https://github.com/Mapotempo/optimizer-api/pull/72)
+- Wrong number of visits [#86](https://github.com/Mapotempo/optimizer-api/pull/86)
+- Protobuf needs binarymode [#87](https://github.com/Mapotempo/optimizer-api/pull/87)
+- Rewind protobuf output before decode [#87](https://github.com/Mapotempo/optimizer-api/pull/87)
+- Parse the last solution provided by optimizer-ortools only once [#87](https://github.com/Mapotempo/optimizer-api/pull/87)
+- Uniformize wrappers output accordingly to API documentation [#83](https://github.com/Mapotempo/optimizer-api/pull/83)
+
+### Deprecated
```

Comment:
Why not "Removed" ?

Reply:
The items are still linked to the features, but we encourage users to change the way they model their problem